### DISCUSSION TITLE
Lisence

### DISCUSSION THREAD

**Discussion by ridwanpandi** [nasirkhan/laravel-starter#299] [General]
hello @nasirkhan, you have a good work for this, I am happy found this, because it's really same with mine but I really interesting with yours since you using Core-UI.
I have question Sir! can I use this starter-kit for my project? what lisence you used?
I can see on your project using this https://github.com/nasirkhan/laravel-starter/blob/master/LICENSE.md is it correct?
thanks in advance!

**Comment by nasirkhan** [comment]
I am happy that you found Laravel Starter useful for your own project. 

You can use this template project and build your product/project upon it. For both commercial and noncommercial use, you should mention the name of the original author and a link back to this repo. 

As you are planning to extend the project, it would be really helpful if you can submit Pull Requests to fix/ improve this project.

**Reply by ridwanpandi** [reply]
thank you @nasirkhan Sir, I really happy for your response. Sure I am going to mention you as contributor of my project since I using this Starter, thanks a lot Sir!
